bundles/AdminBundle/Security/User/TokenStorageUserResolver.php

```diff
@@ -15,28 +15,14 @@
 
 namespace Pimcore\Bundle\AdminBundle\Security\User;
 
-use Pimcore\Bundle\AdminBundle\Security\User\User as UserProxy;
 use Pimcore\Model\User;
-use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+use Pimcore\Bundle\AdminBundle\Security\User\User as UserProxy;
 
 /**
- * Resolves the current pimcore user from the token storage.
+ * @deprecated and will be removed in Pimcore 11. Use \Pimcore\Security\User\TokenStorageUserResolver instead.
  */
-class TokenStorageUserResolver
+class TokenStorageUserResolver extends \Pimcore\Security\User\TokenStorageUserResolver
```

Comment:
I think we can also remove the methods here since, they do not differ from https://github.com/pimcore/pimcore/blob/057a6eb57db6c83ec7b7a3a071b5a5cc8bbd461e/lib/Security/User/TokenStorageUserResolver.php

Reply:
yes, and still we need to keep `getUserProxy` since it has a check on deprecated class https://github.com/pimcore/pimcore/blob/057a6eb57db6c83ec7b7a3a071b5a5cc8bbd461e/bundles/AdminBundle/Security/User/TokenStorageUserResolver.php#L56